plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/github.test.ts

```diff
@@ -997,6 +1010,7 @@ describe('publish:github', () => {
       requireCodeOwnerReviews: false,
       requiredStatusCheckContexts: [],
       requireBranchesToBeUpToDate: true,
+      requiredConversationResolution: false,
```

Comment:
Let's have at least one test in each of these actions that use the new option :)

Reply:
Done.

Reply:
Hmm, are you sure? :) I was expecting at least one test in each asserting `requiredConversationResolution: true`

Reply:
Whoops! Rookie mistake; I didn't actually push the new tests! PTAL